### PR TITLE
Add IMGPROXY_LOG_LEVEL environment config for set Logrus debug level

### PR DESCRIPTION
**Pull request by william-p**
Hi,
all in the title ;)

Fyi, it's my first PR in Go, feedbacks are welcome.

Thanks

### CODE REVIEW

File: log.go [hunk 1]
```diff
@@ -19,7 +19,15 @@ func initLog() {
 		logrus.SetFormatter(newLogPrettyFormatter())
 	}
 
-	logrus.SetLevel(logrus.DebugLevel)
+	logLevel := "debug"
```
Comment (DarthSim) [inline log.go]: Let's make it `info` by default. I made it `debug` because it wasn't configurable.

File: docs/configuration.md [hunk 2]
```diff
@@ -223,6 +223,8 @@ imgproxy can report occurred errors to Bugsnag, Honeybadger and Sentry:
   * `structured`: machine-readable format;
   * `json`: JSON format;
```
Comment (DarthSim) [inline docs/configuration.md]: Remove this empty line please

**Comment by DarthSim** [comment]
Hi,

Thanks for your contribution! It almost LGTM. Fix the things from my comments, and I'll merge this.

**Comment by william-p** [comment]
Done :)

**Comment by DarthSim** [comment]
Thanks again for your contribution!

**Pull request merged**